templates/components/_nav_desktop.html.twig

```diff
@@ -165,7 +162,11 @@
             </div>
             {% if (not app.user) %}
                 <a href="{{ path('app_user_login') }}" class="text--body link--no-decor b__nudge--right">Connexion</a>
-                <a href="{{ path('app_membership_register') }}" class="btn btn--blue">Inscription</a>
+                <a href="{{ path('app_membership_join') }}" class="text--body b__nudge--right">Adhérer</a>
+                <a href="{{ path('app_membership_join') }}" class="btn btn--pink">Donner</a>
+            {% elseif not has_role_adherent and has_role_user %}
+                <a href="{{ path('app_membership_join') }}" class="text--body b__nudge--right">Adhérer</a>
+                <a href="{{ path('app_membership_join') }}" class="btn btn--pink">Donner</a>
```

Comment:
Maybe i am missing something but why does a `Donate` button links to the `app_membership_join` route?

And what about adherents? they shouldn't donate anymore? :fearful: